buoyant_cloud/manifest.json

```
@@ -0,0 +1,94 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dee4b74f-34b7-457e-98b1-7bb8306f2c18",
+  "app_id": "buoyant-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
```

Ah, we need the uninstallation instructions here too:
```suggestion
    "configuration": "README.md#Setup",
    "uninstallation": "README.md#Uninstallation",
```